boosty/utils/json.py

```diff
@@ -1,7 +1,12 @@
 from pathlib import Path
+from types import ModuleType
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    json: ModuleType
 
 try:
-    from orjson import json
```

Comment:
I'm not even sure if using ImportError is worth it. Maybe it's better to organize requirements.txt with necessary dependencies? Anyway, there are no errors from the type checker.